public/logTemplate.js

```diff
@@ -0,0 +1,64 @@
+function createLogTemplate(selectedPilot) {
+	if (!selectedPilot) return;
+
+	var logTemplate = `
+	<div id="logTemplate" class="row logs-header">
+		<h2 class="addon-header">Game Logs</h2>
+	</div>
+	<div class="row">
+		<div id="log-display" class="row">
+			{{#each logs}}
+				<div>
+					<span class="{{user}}_text">[{{ user }}]</span> - {{ message }}
+				</div>
+			{{/each}}
+		</div>
+	</div>
+	<div id="log-input" class="row">
+		<div>
+			<span class="${selectedPilot}_text">${selectedPilot}</span>
+			<span>:~$ </span>
+		</div>
+		<div>
+			<input id="log-text-input" type="text"/>
+		</div>
+	</div>`;
+
+	return logTemplate;
+}
+
+async function sendMessage(e) {
+	if (e.code == "Enter" || e.code == "NumpadEnter"){
+		const message = {
+			user: parent.state.selectedPilot,
+			time: Date.now(),
+			message: this.value
```

Comment:
I feel like I'd want to use `e.target.value` here, because the `this` is a vague ambiguous variable I dislike in JS, but its presumably the same thing and works anyway :shrug: 
```suggestion
			message: e.target.value,
```